src/constants/constants.ts

```diff
@@ -13,8 +13,7 @@
 */
 
 /* eslint-disable @typescript-eslint/ban-types */
```

Comment:
This comment above is for the whole file? Or the next line? If for next line, might be irrelevant and can be deleted

Reply:
Doesn't seem necessary.